### ISSUE TITLE
FontAndColors setting per textMate registration doesn't work

### ISSUE THREAD

**Issue by albilu** [apache/netbeans#5941]
### Apache NetBeans version

Apache NetBeans 17

### What happened

When changing colors for a registered TextMate Language it will not apply to the specific language. (Figure 1)

But when the color changes are done via TextMate Languages it works.But the drawbacks is that the changes apply to all registered TextMate Languages.  (Figure 2)

**We should be able to change FontAndColor for one specific registered TextMate Language not for all at once**

Figure 1:
![image](https://github.com/apache/netbeans/assets/35330562/6fd52233-9233-4995-9275-6a497aeeccd9)
Figure 2:
![image](https://github.com/apache/netbeans/assets/35330562/3bfc1443-7302-43db-ac80-fa81571f47a0)

### How to reproduce

Steps to reproduce: 
1- Registered FontsAndColors in the layer.xml for a specific TextMate Language
2- Use the FontANdColor setting to change color for say Language
3- Changes doesnt apply

Reference: https://github.com/albilu/netbeansPython/discussions/34

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Any

### JDK

Any

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

**Comment by Chris2011** [comment]
Yes, TypeScript is also a Textmate language. Maybe we need a new registration process for this?
